docs/reference/changelog.mdx

```diff
@@ -114,7 +124,7 @@ the API key/secret provided.
 This is one of many PRs that have improved out metric suite to include a bunch
 of new metrics. Some of my personal favorites:
 - Metrics around the current kafka streams state (`RUNNING`, `REBALANCING`, etc...)
-- Some metrics around the Responsive Cache effecitveness
+- Some metrics around the Responsive Cache effectiveness
```

Comment:
Is it worth mentioning any of the scylla/mongo metrics we added?

Reply:
discussed this offline and we decided to leave this out since the metrics are not exposed to users at this point